src/2/README.md

```diff
@@ -1,6 +1,6 @@
 # Beginner
 
-- [Sending Messages](send-msg)
+- [Sending Messages](send-msg/README.md)
```

Comment:
This was left out intentionally. Check generated HTML pages after this change. It gives you a 404. But having just `dir-name/` maps to `dir-name/index.html`

Reply:
I'll update the validator to count directories with readme files as entities too.
The markdown extension I'm using on VS doesn't like the missing readme.